### DISCUSSION TITLE
Sphinx 8.0.0 candidate 1

### DISCUSSION THREAD

**Discussion by AA-Turner** [sphinx-doc/sphinx#12682] [Project Announcements]
The first release candidate for Sphinx 8 has been released and can be found on PyPI at [Sphinx 8.0.0rc1](https://pypi.org/project/Sphinx/8.0.0rc1/).

Please test this candidate, especially if you maintain an extension or theme. Sphinx 8 has several incompatible changes and removes several features that were deprecated in prior versions of Sphinx. Notably, support for Python 3.9 has been removed. See the [detailed changelog](https://www.sphinx-doc.org/en/master/changes.html) or the [tracking issue](https://github.com/sphinx-doc/sphinx/issues/12629) for further information.

To test the release candidate, run ``pip install Sphinx==8.0.0rc1``

Should you encounter any issues, please write here or on [GitHub issues](https://github.com/sphinx-doc/sphinx/issues) with further detail.

Regards from cloudy Britain,
Adam

**Comment by kaycebasques** [comment]
Link to the hashes for anyone else that needs them: https://pypi.org/project/Sphinx/8.0.0rc1/#copy-hash-modal-29fd11a5-8839-442e-a306-1bd565e983a6

**Comment by kaycebasques** [comment]
I tried to see if https://pigweed.dev is OK but can't because Furo seems to depend on Sphinx being between v6 and v8

```
The conflict is caused by:
    furo 2024.7.18 depends on sphinx<8.0 and >=6.0
    The user requested (constraint) sphinx==8.0.0rc1
```

cc @pradyunsg

**Reply by pradyunsg** [reply]
You'll need to use `main`. :)

`pip install https://github.com/pradyunsg/furo/archive/refs/heads/main.zip`

**Comment by Carreau** [comment]
I build the IPython docs with 8.0.0rc1, and I haven't encountered build errors. The output docs looks ok at first glance.

> Regards from cloudy Britain,

Hello from across the British channel where we can see you when the wether is clear; that is to say a few seconds several times a day.
